docs/run_test/quick_run.md

```diff
@@ -0,0 +1,90 @@
+# Getting started with Azure
```

Comment:
It looks only covering Azure? How about ready and local?

Reply:
This serves as a guide for starters, and I think it may be the easiest to get started with Azure, so only Azure was covered. I will improve how to run on other platforms in platform.md if feasible.

Reply:
We can put local run (`lisa`) in installation document to verify installation. And here, we need to introduce ready installation at least. Some users may start from an existing environment, not azure deployment.